lib/script-loader.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Overrides the script-loader.php file.
+ *
+ * @package gutenberg
+ */
+
+// Remove core actions to override.
+remove_action( 'wp_enqueue_scripts', 'wp_enqueue_global_styles' );
```

Comment:
So, besides moving things around, we are also removing core callbacks and hooking only gutenberg's. That's great. We'll probably need to do it for a few other things: `wp_enqueue_global_styles_custom_css` and `wp_enqueue_global_styles_css_custom_properties`.